sos_brazil/settings.py

```diff
@@ -20,7 +20,6 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-w^n#ym+vb40^ue@3#3q($i%_$_-onxs$ou%=cpagdyjsw7m$0_"
```

Comment:
Shouldn't you put `SECRET_KEY=os.getenv("SECRET_KEY")` instead of just deleting it? I haven't configured it like this yet so i really don't know

Reply:
Done!